run.py

```diff
@@ -2,11 +2,11 @@
 Running the application
 """
 import os
-
 from app import create_app
 
 config_name = os.getenv('APP_SETTINGS')  # config_name = "development"
 app = create_app(config_name)
 
 if __name__ == '__main__':
-    app.run()
+    port = int(os.environ.get('PORT', 5000))
```

Comment:
Is there any reason why this port is specifically 5000 ?  Otherwise good job PR looks good to go.

Reply:
Yeah, there is. When I host the app on Heroku, it has issues binding to a port. The solution I found was to specify one manually, otherwise, Heroku will give a timeout error